cime_config/cesm/machines/buildlib.csm_share

```diff
@@ -26,6 +26,7 @@ my $NINST_LND		= `./xmlquery NINST_LND		-value`;
 my $NINST_OCN		= `./xmlquery NINST_OCN		-value`;
 my $NINST_ROF		= `./xmlquery NINST_ROF		-value`;
 my $NINST_WAV		= `./xmlquery NINST_WAV		-value`;
+my $NINST_ESP		= `./xmlquery NINST_ESP		-value`;
```

Comment:
I do not see any reason to restrict it. While the immediate goal is DART (an ensemble filter), future applications, such as geoengineering, may have the same number of instances as the atmosphere.